pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -39,19 +39,36 @@ in stdenvNoCC.mkDerivation {
     then
       [ undmg darwin.cctools ]
     else
-      [ rpmextract ];
+      [ rpm cpio ];
```

Comment:
What is the advantage of using rpm and cpio over using rpmextract?

Reply:
The `intel-mkl-core-${rpm-ver}.x86_64.rpm` package contains the `.pc` files and some static libraries. If I use `rpm2cpio` and pipe its output to `cpio` I can avoid unpacking the static libraries (and the corresponding `.pc` files) when `enableStatic` is `false`. Otherwise no advantage, it's actually more verbose...

Reply:
At first glance, it would seem simpler/cleaner to leave the `rpmextract`, and instead put the whitelisting of pcfiles in the `installPhase`.  

There's already a glob on the find to find the pcfiles, so we just have to change the glob there to make it vary based on whether you have static libs enabled.